app/meldingen/incident/components/IncidentForm/components/DescriptionWithClassificationInput/index.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import ErrorMessage from '../ErrorMessage/';
+
+const DescriptionWithClassificationInput = ({ handler, touched, hasError, meta, parent }) => (
+  <div>
+    <textarea
+      placeholder={meta.placeholder}
+      onKeyUp={(e) => parent.meta.getClassification(e.target.value)}
```

Comment:
parent.meta? what does this do? Can't you just pass along this function directly as a prop?